packages/web-app-files/src/components/AppBar/SelectedResources/SizeInfo.vue

```diff
@@ -2,7 +2,7 @@
   <div class="uk-flex uk-flex-middle">
     <translate
       v-if="selectedResourcesSize !== '?'"
-      key="multiple-select-info"
+      key="multiple-select-info-with-size"
```

Comment:
this section shows number of selected along with size so I think the key should be `multiple-select-info-with-size`